schema-library-events/src/main/avro/connectorConfig.avdl

```diff
@@ -0,0 +1,36 @@
+@namespace("com.homeaway.digitalplatform.streamregistry")
+
+protocol ConnectorConfigProtocol {
+
+  /**
+  * S3 Connector gets data from Kafka and dumps it in S3
```

Comment:
This isn't specific to S3.

Reply:
I agree. My bad, it was a copy-over error.